gitbook/src/help/troubleshooting-steps.md

```diff
@@ -0,0 +1,24 @@
+# Troubleshooting
+
+When an issue arises, follow these steps to methodically diagnose and resolve the problem.
+
+1. **Check Hardware Requirements:** The most common cause of issues is insufficient disk or RAM. Ensure your system meets the recommended specifications.  
```

Comment:
Erigon is very adaptive - adding more RAM to server will make Erigon faster (without any settings change)